Gruntfile.js

```diff
@@ -0,0 +1,1259 @@
+// Gruntfile for uProxy
+/* jshint node: true */
+/* jshint esversion: 6 */
+
+module.exports = function(grunt) {
+  'use strict';
+
+  require('time-grunt')(grunt);
+  const _ = require('lodash');
+  const fs = require('fs');
+  const rules = require('./build/tools/common-grunt-rules');
+  const path = require('path');
+
+//-------------------------------------------------------------------------
+
+  // Location of where src is copied into and compiled
+  const devBuildPath = 'build/src';
+  const distBuildPath = 'build/dist';
+  // Location of where to copy/build third_party source/libs.
+  const thirdPartyBuildPath = 'build/third_party';
+  // This is used for the copying of uproxy libraries into the target directory.
+  const localLibsDestPath = '';
```

Comment:
Weird, looks like this isn't used:
https://github.com/uProxy/uproxy/issues/2729